graphql/pupil/mutations.ts

```diff
@@ -50,6 +50,28 @@ export class MutatePupilResolver {
         return true;
     }
 
+    @Mutation(returns => Boolean)
+    @Authorized(Role.ADMIN)
```

Comment:
Is this endpoint supposed to be restricted to admins? Shouldn't a pupil be able to join the waiting list themselves? Or is this an admin-only endpoint and there's a different one for pupils?

Reply:
Yeah, we currently do not have user authentication for the GraphQL endpoint. If you have time you could also have a look at the "GraphQL Authentication" Pull Request which adds exactly that ...